src/Command/Cache/DoctrineResultCacheWarmerCommand.php

```diff
@@ -224,13 +221,17 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $progressBar->setMaxSteps($numberOfTotalUsers);
         $progressBar->start();
 
+        $lastUserId = 0;
+        $numberOfWarmedUpCacheEntries = 0;
         do {
             $iterateResult = $this
-                ->getFindAllUsernameQuery($offset)
+                ->getFindAllUsernameQuery($lastUserId)
                 ->iterate();
 
             foreach ($iterateResult as $row) {
-                $this->warmUpResultCacheForUserName(current($row)['username']);
+                $username = current($row)['username'];
```

Comment:
Are these current() calls really needed?

Reply:
As I see - they are. But, what I would propose is to move the `curent()` call to an additional variable:
```
$currentRow = current($row)
```
and then just
```
$username = $currentRow['username'];
```